fuzz/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY . .
 
 RUN dnf install -q -y make clang golang && \
     update-alternatives --install /usr/bin/cc cc /usr/bin/clang 20 && \
-    go install github.com/mdempsky/go114-fuzz-build@latest && \
+    go install github.com/mdempsky/go114-fuzz-build@v0.0.0-20200813011514-561a369ae6e1 && \
```

Comment:
We use @latest for the tools in the main Makefile https://github.com/zalando/skipper/blob/2fd26a2d1784a6942f64d33d463f7366ae848e21/Makefile#L118-L122 and I think it makes sense to use latest here when this image is built

Reply:
Makefile choices does not matter for the scanner

Reply:
I'll approve and bet we'll forget to update this version as there is no automatic update.

Reply:
@AlexanderYastrebov although I agree with you, but this tool doesn't change that much (or even at all, look at the last release), this is because this is just a code generator for clang.